### PR TITLE
Pick up VS Code theme, fixes #228

### LINKED ISSUE #228: Graph does not follow VSCode theme

### PR DESCRIPTION
**Pull request by StephenWeatherford**
There will still be some adjustments after meeting with UX, but this handles the major changes.

image

### CODE REVIEW

File: resources/graphClient/graphClient.css [hunk 1]
```diff
@@ -12,12 +12,26 @@
 html {
```
Comment (EricJizbaMSFT) [inline resources/graphClient/graphClient.css]: The colors in the text box seem backwards. For example this is what it looks like in dark theme:
screen shot 2017-12-08 at 3 50 37 pm

But This is what the colors look like when I'm editing a normal file (Notice the _text_ is the light color and the background is dark):
screen shot 2017-12-08 at 3 55 24 pm

File: resources/graphClient/graphClient.css [hunk 2]
```diff
@@ -12,12 +12,26 @@
...
     width: 100%;
```
Comment (EricJizbaMSFT) [inline resources/graphClient/graphClient.css]: It's hard to read 'No data is loaded' text in certain themes:
screen shot 2017-12-08 at 3 51 20 pm

File: resources/graphClient/graphClient.css (hunk 1 above)
Comment (StephenWeatherford) [reply resources/graphClient/graphClient.css]: It's intentional based on the original design from UX. Agreed it doesn't look as good with this theme.  I'll be discussing stuff like that next week and can adjust.

**Comment by StephenWeatherford** [comment]
I’m aware of it, yes.  Not sure of an easy fix, but the work-around is pretty easy so I’m not worrying about it now.

**Pull request merged**